tests/queries/0_stateless/02346_text_index_preprocessor.sql

```diff
@@ -0,0 +1,73 @@
+SET allow_experimental_full_text_index = 1;
+SET use_skip_indexes_on_data_read = 1;
+
+-- Basic test preprocessor clause usage with lower
```

Comment:
Minor: `Tests the preprocessor argument for tokenizers in the text index definitions.`